backend/src/seeds/seed.vehicles.ts

```diff
@@ -0,0 +1,27 @@
+import { Vehicle } from '../vehicles/entities/vehicle.entity';
+import { Factory } from './seed.utils';
+import { organizationsFactory } from './seed.organization';
+
+class VehicleFactory extends Factory<Vehicle> {
+  constructor() {
+    super('Vehicle');
+  }
+
+  public generate() {
+    const vehicle = new Vehicle();
+    vehicle.licensePlate = this.faker.vehicle.vrm();
+    vehicle.notes = this.faker.random.words(10);
```

Comment:
Maybe we could also randomize if the note has a note, or not? 
```suggestion
    vehicle.notes = this.faker.datatype.number({ max: 1, min: 0, precision: 0.0000001 }) < 0.2 ? this.faker.random.words(10) : undefined;
```

We're comparing to 0.2 to have a 20% chance of having a note